src/main/resources/db/migration/V1_5__create_courthouse_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE moj_courthouse (
+  moj_crt_id                INTEGER                         GENERATED ALWAYS AS IDENTITY
```

Comment:
Please remove the IDENTITY stuff - the Java code only had that because at the time we didn't have any SEQUENCEs in Barry's v5 sql

`moj_crt_id                INTEGER					 NOT NULL`